Dockerfile

```diff
@@ -137,199 +137,197 @@ RUN apt-get install -y libgl1-mesa-glx && \
+        google-cloud-language==2.* \
+        google-cloud-videointelligence==2.* \
+        google-cloud-vision==2.* \
+        # b/183041606#comment5: the Kaggle data proxy doesn't support these APIs. If the library is missing, it falls back to using a regular BigQuery query to fetch data.
+        pip uninstall -y google-cloud-bigquery-storage \
+        # After launch this should be installed from pip
+        git+https://github.com/googleapis/python-aiplatform.git@mb-release \ 
+        ortools \
+        scattertext \
+        # Pandas data reader
+        pandas-datareader \
+        wordsegment \
+        wordbatch \
+        emoji \
+        # Add Japanese morphological analysis engine
+        janome \
+        wfdb \
+        vecstack \
+        # yellowbrick machine learning visualization library
+        yellowbrick \
+        mlcrate && \
     /tmp/clean-layer.sh
 
-RUN pip install bleach && \
-    pip install certifi && \
-    pip install cycler && \
-    pip install decorator && \
-    pip install entrypoints && \
-    pip install html5lib && \
-    pip install ipykernel && \
-    pip install ipython && \
-    pip install ipython-genutils && \
-    pip install ipywidgets && \
-    pip install isoweek && \
-    pip install jedi && \
-    pip install Jinja2 && \
-    pip install jsonschema && \
-    pip install jupyter-client && \
-    pip install jupyter-console && \
-    pip install jupyter-core && \
-    pip install MarkupSafe && \
-    pip install mistune && \
-    pip install nbconvert && \
-    pip install nbformat && \
-    pip install notebook && \
-    pip install papermill && \
-    pip install olefile && \
-    # b/198300835 kornia 0.5.10 is not compatible with our version of numpy.
-    pip install kornia==0.5.8 && \
-    pip install pandas_summary && \
-    pip install pandocfilters && \
-    pip install pexpect && \
-    pip install pickleshare && \
-    pip install Pillow && \
-    # Install openslide and its python binding
-    apt-get install -y openslide-tools && \
-    pip install openslide-python && \
-    pip install ptyprocess && \
-    pip install Pygments && \
-    pip install pyparsing && \
-    pip install pytz && \
-    pip install PyYAML && \
-    pip install pyzmq && \
-    pip install qtconsole && \
-    pip install six && \
-    pip install terminado && \
-    pip install tornado && \
-    pip install tqdm && \
-    pip install traitlets && \
-    pip install wcwidth && \
-    pip install webencodings && \
-    pip install widgetsnbextension && \
-    pip install pyarrow && \
-    pip install feather-format && \
-    # fastai >= 2.3.1 upgrades pytorch/torchvision. upgrade of pytorch will be handled in b/181966788
-    pip install fastai==2.2.7 && \
-    pip install allennlp && \
-    # https://b.corp.google.com/issues/184685619#comment9: 3.9.0 is causing a major performance degradation with spacy 2.3.5
-    pip install importlib-metadata==3.4.0 && \
+RUN # Install openslide and its python binding
```

Comment:
You should move the comment above the `RUN` command.

You get the following error: `Error response from daemon: dockerfile parse error line 277: unknown instruction: APT-GET`

You can build it locally by running:

```
cd /path/to/docker-python
/build -c
```